bedrock/urls.py

```diff
@@ -34,6 +34,7 @@
     (r'^styleguide/', include('bedrock.styleguide.urls')),
     (r'^tabzilla/', include('bedrock.tabzilla.urls')),
     (r'^security/', include('bedrock.security.urls')),
+    (r'^shape_of_the_web/', include('bedrock.shapeoftheweb.urls')),
```

Comment:
Mostly for consistency I think so. I'm not going to r- about it though, so I leave it to you. That's just my feeling, not a rule or anything.